### PR TITLE
Add two notes about the job

### PR DESCRIPTION
**Pull request by choldgraf**
After some feedback about our job posting, this adds two extra sets of content:

1. Defines the timeline for application
2. Adds a note that we recognize the large scope of this role, and shows our commitment to making this role sustainable for the person involved.

### CODE REVIEW

File: content/jobs/2022/product-community-lead.md [hunk 1]
```diff
@@ -82,4 +83,10 @@ Below are several skills that will make somebody well-suited for this role. Ther
...
-* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
\ No newline at end of file
+* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
+
+## A note on scope and expectations
+
+We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c.
+
+We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable.
```
Comment (choldgraf) [inline content/jobs/2022/product-community-lead.md]:
```suggestion
We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable. The person in this role will need to be comfortable with a larger-than-normal amount of ambiguity and flexibility, though part of their job will be to _reduce_ this ambiguity for themselves and for others in the future.
```

File: content/jobs/2022/product-community-lead.md [hunk 2]
```diff
@@ -82,4 +83,10 @@ Below are several skills that will make somebody well-suited for this role. Ther
...
-* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
\ No newline at end of file
+* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
+
+## A note on scope and expectations
+
+We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c.
+
+We recognize that one risk in these situations is to have unrealistic expectations, or an unsustainable amount of work for the person involved. We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to ensure that this role is sustainable. The person in this role will need to be comfortable with a larger-than-normal amount of ambiguity and flexibility, though part of their job will be to _reduce_ this ambiguity for themselves and for others in the future.
```
Comment (damianavila) [inline content/jobs/2022/product-community-lead.md]: I recognize the intention to be explicit about the risk involved but IMHO being that explicitness goes against the job posting attractiveness. 
I would try to be more straightforward, instead. I mean... the first sentence looks great (line 90) and then I will just follow up just with "We'll do everything we can to set these expectations intentionally, and to refine them as we learn more, to _reduce_ this ambiguity, to ensure that this role is sustainable."
Just my totally subjective point of view 😉.

Comment (choldgraf) [inline content/jobs/2022/product-community-lead.md]:
```suggestion
We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c. We will do everything we can to set these expectations intentionally, and to refine them as we learn more, in order to reduce this ambiguity and ensure that this role is sustainable.
```

File: content/jobs/2022/product-community-lead.md [hunk 3]
```diff
@@ -82,4 +83,8 @@ Below are several skills that will make somebody well-suited for this role. Ther
...
 * Experience or strong interest in **cloud infrastructure and data workflows** for research or education, and how this can lead to more collaboration, openness, and productivity (for example, via managed services and [Analysis Ready, Cloud Optimized datasets](https://eartharxiv.org/repository/view/2726/)).
-* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
\ No newline at end of file
+* Experience with **distributed and complex open source communities** and ability to communicate and collaborate effectively with these communities.
+
+## A note on scope and expectations
+
+We know that this role is very broad in scope, and that there is a lot of room for interpretation of the specific responsibilities and duties. This is somewhat intentional, and somewhat unavoidable, given that this is the first role of its kind within 2i2c. We will do everything we can to set these expectations intentionally, and to refine them as we learn more, in order to reduce this ambiguity and ensure that this role is sustainable.
```
Comment (choldgraf) [inline content/jobs/2022/product-community-lead.md:90]: What do you think @damianavila - good point about not under-selling ourselves :-)

**Comment by choldgraf** [comment]
I think I've addressed @damianavila's comment, so will merge this one now since the page is live

File: content/jobs/2022/product-community-lead.md (hunk 3 above)
Comment (damianavila) [reply content/jobs/2022/product-community-lead.md:90]: Super! Thanks!

**Pull request merged**